org.eclipse.jdt.core.tests.builder/src/org/eclipse/jdt/core/tests/builder/DependencyTests.java

```diff
@@ -1149,9 +1150,9 @@ public void testTypeVisibility2() throws JavaModelException {
 	}
 
 	public void testTypeVariable() throws JavaModelException {
-		if ((AbstractCompilerTest.getPossibleComplianceLevels() & AbstractCompilerTest.F_1_5) == 0) return;
+		if ((AbstractCompilerTest.getPossibleComplianceLevels() & AbstractCompilerTest.FIRST_SUPPORTED_JAVA_VERSION) == 0) return;
```

Comment:
I would get rid of this line - The old code was checking that we are not testing this 1.5 feature in 1.4 mode or below. We don't have that possibility now,